pkgs/development/python-modules/Cython/default.nix

```diff
@@ -11,7 +11,17 @@
 , ncurses
 }:
 
-buildPythonPackage rec {
+let
+  excludedTests = []
+    # cython's testsuite is not working very well with libc++
+    # We are however optimistic about things outside of testsuite still working
+    ++ stdenv.lib.optionals (stdenv.cc.isClang or false) [ "cpdef_extern_func" "libcpp_algo" ]
+    # Some tests in the test suite isn't working on aarch64. Disable them for
+    # now until upstream finds a workaround.
+    ++ stdenv.lib.optionals stdenv.isAarch64 [ "test_coerce_to_numpy" ]
```

Comment:
Please add a link to the upstream issue.

Reply:
Done.